pkg/controller/volume/persistentvolume/pv_controller_base.go

```diff
@@ -398,6 +398,31 @@ func (ctrl *PersistentVolumeController) claimWorker() {
 	}
 }
 
+// resync supplements short resync period of shared informers - we don't want
```

Comment:
Just for my understanding, is resync required due to the informer missing events?  Is it considered a bug or is it "by design"?